### PR TITLE
docs: Ben Sucks

### PR DESCRIPTION
**Pull request by carbonaratara**

### CODE REVIEW

**Review by bpmct — CHANGES_REQUESTED** [review]
no!

File: docs/about/architecture.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This document provides a high level overview of Coder's architecture.
 
 coderd is the service created by running `coder server`. It is a thin API that
 connects workspaces, provisioners and users. coderd stores its state in Postgres
-and is the only service that communicates with Postgres.
+and is the only service that communicates with Postgres.Ben Sucks.
```
Comment (bpmct) [inline docs/about/architecture.md:13]: let's make this more positive

**Comment by bpmct** [comment]
( this was from an internal workshop on how to contribute to our docs )

we're all friends here at Coder :)

File: docs/about/architecture.md (hunk 1 above)
Comment (johnstcn) [reply docs/about/architecture.md:13]:
```suggestion
and is the only service that communicates with Postgres.Ben creates negative pressure.
```

**Pull request closed without merging**